### DISCUSSION TITLE
Where cain I see password of martel@kitt4sme.collab-cloud.eu?

### DISCUSSION THREAD

**Discussion by alvaroo278** [c0c0n3/kitt4sme.live#189] [General]
I can't see password con VM

**Comment by c0c0n3** [comment]
Hi @alvaroo278,

We can't share that password w/ you, the kitt4sme.collab-cloud.eu cluster is only supposed to be accessed by Kitt4sme core devs :-)

But you can assemble your own kitt4sme instance by following the cluster bootstrap procedure
- https://github.com/c0c0n3/kitt4sme.live/blob/main/docs/bootstrap.md
